themes/gitline/powerline.base.bash

```diff
@@ -0,0 +1,234 @@
+# Define this here so it can be used by all of the Powerline themes
```

Comment:
Do you need this file here? Looks like a copy/paste. I don't think you're using this file in the new theme, right?

Please remove the file...

Reply:
Yes I used the redline theme as the base file to work on, I will remove all the unnecessary and redundant data in the code.

Reply:
Sems like I need to define that variable after every command to check the sudo status of the user.
My theme changes the part of the users name when he goes sudo and adds a crown emoticon before it. Like this:

Reply:
I need the powerline base file for the SCM prompt characters
the gitline file is for the colors

Reply:
It doesn't look like you're loading the `powerline.base.bash` file - your commit shows that you're still loading the one from the `redline` theme...